### PR TITLE
Convert hello examples to use new APIs

### PR DESCRIPTION
**Pull request by nvkevlu**
Convert hello examples to use new APIs

### Description

Convert hello examples to use new APIs.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: examples/hello-world/hello-cyclic/fedavg_script_executor_hello-cyclic.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    # Define the initial global model and send to server
+    job.to(Net(), "server")
+
+    # Add clients
+    for i in range(n_clients):
+        executor = ScriptExecutor(
+            task_script_path=train_script,
+            task_script_args="",
+            params_exchange_format=ExchangeFormat.NUMPY,  # f"--batch_size 32 --data_path /tmp/data/site-{i}"
```
Comment (SYangster) [inline examples/hello-world/hello-cyclic/fedavg_script_executor_hello-cyclic.py:43]: move comment to task_script_args line

File: examples/hello-world/hello-pt/nvflare_pt_getting_started.ipynb [hunk 2]
```diff
@@ -0,0 +1,412 @@
+{
```
[the remark this answers is not in the document]
Comment (nvkevlu) [reply examples/hello-world/hello-pt/nvflare_pt_getting_started.ipynb:1]: I don't think it is meaningfully different, so I will remove this and link instead.

**Comment by nvkevlu** [comment]
Using this PR instead: https://github.com/NVIDIA/NVFlare/pull/2785

**Pull request closed without merging**